apps/backend-infrastructure/lib/ecs-service.ts

```diff
@@ -131,6 +154,12 @@ export class ECSService extends NestedStack {
             FLOOD_ASSET_ID: ecs_Secret.fromSecretsManager(floodAssetId),
             INCIDENT_ASSET_ID: ecs_Secret.fromSecretsManager(incidentAssetId),
             DROUGHT_ASSET_ID: ecs_Secret.fromSecretsManager(droughtAssetId),
+            WEBHOOK_TOKEN: ecs_Secret.fromSecretsManager(webhookToken),
+            TELEGRAM_BOT_TOKEN: ecs_Secret.fromSecretsManager(telegramBotToken),
+            TELEGRAM_PCDM_CHAT_ID:
+              ecs_Secret.fromSecretsManager(telegramPcdmChatId),
+            TELEGRAM_NCDM_CHAT_ID:
+              ecs_Secret.fromSecretsManager(telegramNcdmChatId),
```

Comment:
Do they all need to be secrets?

Reply:
We could refactor to have the chat ids somewhere else

Reply:
Can a chat id be used without the proper token?
If I create a random telegram token can I start spamming this chat? Or do the token and chat_id need to be linked somehow?